cloudslang-entities/src/main/java/io/cloudslang/lang/entities/LoopStatement.java

```diff
@@ -37,4 +40,20 @@ protected LoopStatement() {
     public String getExpression() {
         return expression;
     }
+
```

Comment:
Why do we use these libraries? they use reflection to determine the fields of each object. It is probably slower than doing it manually.